src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -594,6 +607,21 @@ private List<LaunchCommand> GetInitializeCommands()
                         commands.Add(new LaunchCommand("-gdb-set new-console on", ignoreFailures:true));
                     }
 
+                    // If running locally on windows, determine if gdb is running from cygwin
+                    if (localLaunchOptions != null &&
```

Comment:
I was originally thinking no because I'll eventually need a way to do this on lldb, but lldb won't support this command so I guess yes.